autoload/go/textobj.vim

```diff
@@ -60,6 +64,16 @@ function! go#textobj#Function(mode) abort
     " want's to include doc comments for function declarations
     if has_key(info, 'doc') && g:go_textobj_include_function_doc
       call cursor(info.doc.line, info.doc.col)
+    elseif info['sig']['name'] == '' && g:go_textobj_include_variable
```

Comment:
I think you meant `g:go_textobj_include_function_assignment` here, and not `g:go_textobj_include_variable`?